README.md

```diff
@@ -21,6 +21,11 @@ just run `docker swarm init`.
 It's also necessary to manually load the IPVS kernel module because as swarms are created in `dind`, 
 the daemon won't load it automatically. Run the following command for that purpose: `sudo lsmod xt_ipvs`
 
+If you want to experiment with a stable version of Docker 1.12 then you can override the requirement for Docker 1.13-dev.
```

Comment:
Shall we clarify that the limitation here is that as we're not using overlay, the amount of networks that can be created reduced?.